nessie/src/main/java/org/apache/iceberg/nessie/NessieTableOperations.java

```diff
@@ -139,26 +131,11 @@ public FileIO io() {
     return fileIO;
   }
 
-  /**
-   * try and get a Spark application id if one exists.
-   *
-   * <p>
-   *   We haven't figured out a general way to pass commit messages through to the Nessie committer yet.
-   *   This is hacky but gets the job done until we can have a more complete commit/audit log.
-   * </p>
-   */
-  private String applicationId() {
-    String appId = null;
-    TableMetadata current = current();
-    if (current != null) {
-      Snapshot snapshot = current.currentSnapshot();
-      if (snapshot != null) {
-        Map<String, String> summary = snapshot.summary();
-        appId = summary.get("spark.app.id");
-      }
-
-    }
-    return appId;
+  Map<String, String> getCatalogOptions() {
```

Comment:
Iceberg rarely uses `get` in function names because it doesn't add anything. The only reason to use `get` is when the function acts as a getter _and_ the class should meet the expectations of a Java bean for some reason, like Spark interoperability. Otherwise, `get` is typically superfluous (as it is in this case) or there is almost certainly another verb that provides more context for _how_ the value is being produced.

Reply:
given that we're passing `catalogOptions` via the constructor, we can remove that method again